themes/openy_themes/openy_rose/css/styles.css

```diff
@@ -8798,3 +8798,7 @@ input[type="checkbox"]:focus {
 .footer-alerts-list .slick__arrow {
   margin-top: -25px;
 }
+.page-head__main-menu .contextual-region {
```

Comment:
we should make changes in scss files, not in css only, otherwise these would be wiped out during next theme css rebuild

Reply:
Sure, I will update the code.

Reply:
@podarok 
I have updated the code, Please review.

Reply:
@kairamkondarajesh nice, thank you
But we should also keep CSS file after compilation in place
Please fix